docs/README_RECON.md

```diff
@@ -0,0 +1,316 @@
+# Remorph Reconciliation
+
+Remorph Reconciliation is a library that helps to reconcile the data post-data migration or during the data
+migration between the source system and Databricks. Supports multiple types of reconciliation depending on the user
+requirements and data in the source system. It simplifies the migration process by removing the complex part of validating 
+the data from one system to another and enabling the team to have more insights into the migrated data.
+
+* [Remorph Reconciliation](#remorph-reconciliation)
+* [Installation](#installation)
+* [Types of Report Supported](#types-of-report-supported)
+* [Report Type-Flow Chart](#report-type-flow-chart)
+* [Supported Source System](#supported-source-system)
+* [Dependency CLI commands](#dependency-cli-commands)
+* [TABLE Config Elements](#table-config-elements)
+   * [jdbc_reader_options](#jdbc_reader_options)
+   * [column_mapping](#column_mapping)
+   * [transformations](#transformations)
+   * [thresholds](#thresholds)
+   * [filters](#filters)
+   * [Key Considerations](#key-considerations)
+* [Reconciliation Examples](#reconciliation-examples)
+* [Frequently Asked Questions](#frequently-asked-questions)
+   * [Reconciler Questions](#reconciler-questions)
+* [Common Error Codes](#common-error-codes)
+
+## Installation
```

Comment:
To be moved inside main readme and then link to this document.

Reply:
Moved FAQ to main read me